application/src/main/java/org/thingsboard/mqtt/broker/service/install/update/DefaultCacheCleanupService.java

```diff
@@ -49,54 +40,75 @@ public class DefaultCacheCleanupService implements CacheCleanupService {
      * Cleanup caches that can not deserialize anymore due to schema upgrade or data update using sql scripts.
      * Refer to SqlDatabaseUpgradeService and /data/upgrade/*.sql
      * to discover what tables were changed
-     * */
+     *
+     * IMPORTANT NOTE: We are using Redis to persist messages for DEVICE persisted clients,
+     * so calling flushAll may lead to data loss. Ensure proper evaluation and backups before performing this operation.
+     */
     @Override
-    public void clearCache(String fromVersion) throws Exception {
-        switch (fromVersion) {
-            case "1.3.0":
-                log.info("Clearing cache to upgrade from version 1.3.0 to 1.4.0");
-                clearAll();
-                break;
-            case "2.0.0":
```

Comment:
Why is clearing of the cache removed for case "2.0.0" if we support upgrade for 2.0.0 and 2.0.1

private static final List<String> SUPPORTED_VERSIONS_FOR_UPGRADE = List.of("2.0.0", "2.0.1");

Reply:
My bad. At the moment of writing, I have supposed that we will have only one supported version to upgrade. I will fix this ASAP.